SU-HSDO/suhumsci
# SHS-4982: Remove webp conversion from image styles

# READY FOR REVIEW

## Summary
- Remove webp conversion from all image styles

## Need Review By (Date)
- ASAP

## Urgency
- High

## Steps to Test
1. Confirm that the conversion to webp is disabled in all image styles (choose some random ones to test this)
2. Confirm that elements that use a responsive image (e.g. Hero slides) don't have any webp image as an option for the `<picture>` element

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
